# Translations template for cddagl.
# Copyright (C) 2019 ORGANIZATION
# This file is distributed under the same license as the cddagl project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2019.
# 
# Translators:
# KurzedMetal <kurzedmetal@gmail.com>, 2019
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: cddagl 1.4.3\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2019-09-04 12:48-0400\n"
"PO-Revision-Date: 2019-07-10 19:53+0000\n"
"Last-Translator: KurzedMetal <kurzedmetal@gmail.com>, 2019\n"
"Language-Team: Japanese (https://www.transifex.com/cdda-game-launcher/teams/101126/ja/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.6.0\n"
"Language: ja\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#: cddagl/ui/views/backups.py:188
msgid "Backups available"
msgstr "バックアップ使用可能"

#: cddagl/ui/views/backups.py:189
msgid "Manual backup"
msgstr "手動バックアップ"

#: cddagl/ui/views/backups.py:190
msgid "Automatic backups"
msgstr "自動バックアップ"

#: cddagl/ui/views/backups.py:192 cddagl/ui/views/backups.py:316
#: cddagl/ui/views/backups.py:345 cddagl/ui/views/backups.py:633
msgid "Restore backup"
msgstr "バックアップを復元"

#: cddagl/ui/views/backups.py:193
msgid "Refresh list"
msgstr "一覧を更新"

#: cddagl/ui/views/backups.py:194 cddagl/ui/views/backups.py:653
msgid "Delete backup"
msgstr "バックアップを削除"

#: cddagl/ui/views/backups.py:195
msgid "Do not backup the current saves before restoring a backup"
msgstr "バックアップ復元前に現在のセーブをバックアップしない"

#: cddagl/ui/views/backups.py:197
msgid "Name"
msgstr "名前"

#: cddagl/ui/views/backups.py:198
msgid "Modified"
msgstr "変更"

#: cddagl/ui/views/backups.py:198
msgid "Worlds"
msgstr "ワールド"

#: cddagl/ui/views/backups.py:198
msgid "Characters"
msgstr "キャラクター"

#: cddagl/ui/views/backups.py:198
msgid "Actual size"
msgstr "無圧縮サイズ"

#: cddagl/ui/views/backups.py:199
msgid "Compressed size"
msgstr "圧縮後サイズ"

#: cddagl/ui/views/backups.py:199
msgid "Compression ratio"
msgstr "圧縮率"

#: cddagl/ui/views/backups.py:199
msgid "Modified date"
msgstr "変更日"

#: cddagl/ui/views/backups.py:201
#: cddagl/ui/views/soundpacks.py:215
msgid "Name:"
msgstr "名前:"

#: cddagl/ui/views/backups.py:202 cddagl/ui/views/backups.py:1073
msgid "Backup current saves"
msgstr "現在のセーブをバックアップ"

#: cddagl/ui/views/backups.py:204
msgid "Backup saves before game launch"
msgstr "ゲーム起動前にセーブをバックアップ"

#: cddagl/ui/views/backups.py:205
msgid "Backup saves after game end"
msgstr "ゲーム終了後にセーブをバックアップ"

#: cddagl/ui/views/backups.py:207
msgid ""
"This option will only work if you also have the option to keep the launcher "
"opened after launching the game in the settings tab."
msgstr "この設定を使用するには、設定タブの「ゲーム起動後にランチャーを閉じない」を有効にして下さい。"

#: cddagl/ui/views/backups.py:211
msgid "Maximum automatic backups count:"
msgstr "自動バックアップ作成数上限:"

#: cddagl/ui/views/backups.py:314 cddagl/ui/views/backups.py:343
#: cddagl/ui/views/backups.py:375
msgid "Restore backup cancelled"
msgstr "バックアップの復元が中止されました"

#: cddagl/ui/views/backups.py:441 cddagl/ui/views/backups.py:532
msgid "Cancel restore backup"
msgstr "バックアップの復元を中止"

#: cddagl/ui/views/backups.py:473
msgid "Could not rename the save directory"
msgstr "セーブディレクトリをリネーム出来ませんでした"

#: cddagl/ui/views/backups.py:478
msgid "Could not remove the save file"
msgstr "セーブファイルを削除出来ませんでした"

#: cddagl/ui/views/backups.py:493
msgid "Extracting backup"
msgstr "バックアップ展開中"

#: cddagl/ui/views/backups.py:555
msgid "Extracting {filename}"
msgstr "展開中 {filename}"

#: cddagl/ui/views/backups.py:576
msgid "{backup_name} backup restored"
msgstr "{backup_name} バックアップを復元しました"

#: cddagl/ui/views/backups.py:654
msgid "This will delete the backup file. It cannot be undone."
msgstr "バックアップファイルを削除します。この操作は元に戻せません。"

#: cddagl/ui/views/backups.py:656
msgid ""
"Are you sure you want to delete the <strong>{filename}</strong> backup?"
msgstr "<strong>{filename}</strong> のバックアップを削除してよろしいですか？"

#: cddagl/ui/views/backups.py:659
msgid "Delete the backup"
msgstr "バックアップを削除"

#: cddagl/ui/views/backups.py:661
msgid "I want to keep the backup"
msgstr "バックアップを削除しない"

#: cddagl/ui/views/backups.py:670
msgid "Backup deletion cancelled"
msgstr "バックアップ削除を中止しました"

#: cddagl/ui/views/backups.py:675
msgid "Backup deleted"
msgstr "バックアップを削除しました"

#: cddagl/ui/views/backups.py:690 cddagl/ui/views/backups.py:732
msgid "Manual backup cancelled"
msgstr "手動バックアップを中止しました"

#: cddagl/ui/views/backups.py:749 cddagl/ui/views/main.py:378
#: cddagl/ui/views/main.py:527 cddagl/ui/views/main.py:906
msgid "auto"
msgstr "自動"

#: cddagl/ui/views/backups.py:785
msgid "Save directory not found"
msgstr "セーブディレクトリが見つかりません"

#: cddagl/ui/views/backups.py:804
msgid "Could not delete previous backup archive"
msgstr "以前のバックアップアーカイブを削除出来ませんでした"

#: cddagl/ui/views/backups.py:893
msgid "Cancel backup"
msgstr "バックアップを中止"

#: cddagl/ui/views/backups.py:896
msgid "Searching for save files"
msgstr "セーブファイル検索中"

#: cddagl/ui/views/backups.py:907
msgid "Found {filename} in {path}"
msgstr "場所 {path} で {filename} を発見しました"

#: cddagl/ui/views/backups.py:926
msgid "Compressing save files"
msgstr "セーブファイル圧縮中"

#: cddagl/ui/views/backups.py:993
msgid "Compressing {filename}"
msgstr "圧縮中 {filename}"

#: cddagl/ui/views/backups.py:1015
msgid "Saves backup completed"
msgstr "セーブのバックアップが完了しました"

#: cddagl/ui/views/backups.py:1279
msgid "Cannot rename file"
msgstr "ファイルをリネームできません"

#: cddagl/ui/views/backups.py:1285
msgid ""
"\n"
"<p>The launcher failed to rename the following file: {src} to {dst}</p>\n"
"<p>When trying to rename or access {filename}, the launcher raised the\n"
"following error: {error}</p>\n"
msgstr ""
"\n"
"<p>以下のファイルのリネームに失敗しました: {src} to {dst}</p>\n"
"<p>{filename}へのアクセスまたはリネームを試みましたがエラーが発生しました\n"
": {error}</p>\n"

#: cddagl/ui/views/backups.py:1296
msgid ""
"\n"
"<p>No process seems to be using that file.</p>\n"
msgstr ""
"\n"
"<p>指定されたファイルを使用中のプロセスはありません。</p>\n"

#: cddagl/ui/views/backups.py:1300
msgid ""
"\n"
"<p>The process <strong>{image_file_name} ({pid})</strong> is currently using\n"
"that file. You might need to end it if you want to retry.</p>\n"
msgstr ""
"\n"
"<p>プロセス <strong>{image_file_name} ({pid})</strong> が、指定された\n"
"ファイルを使用中です。上記プロセス終了後に再試行して下さい。</p>\n"

#: cddagl/ui/views/backups.py:1306
msgid "Do you want to retry renaming this file?"
msgstr "ファイルのリネームを再試行しますか？"

#: cddagl/ui/views/backups.py:1308
msgid "Retry renaming the file"
msgstr "ファイルのリネームを再試行"

#: cddagl/ui/views/backups.py:1310 cddagl/ui/views/main.py:3250
#: cddagl/ui/views/main.py:3312
msgid "Cancel the operation"
msgstr "中止"

#: cddagl/ui/views/dialogs.py:37
msgid ""
"This {name} cannot be directly downloaded by the launcher. You have to use "
"your browser to download it."
msgstr "{name}はランチャーから直接ダウンロード出来ません。ブラウザからダウンロードして下さい。"

#: cddagl/ui/views/dialogs.py:44
msgid "1. Open the URL in your browser."
msgstr "1. URLをブラウザで開く。"

#: cddagl/ui/views/dialogs.py:59
msgid ""
"2. Download the {name} on that page and wait for the download to complete."
msgstr "2. {name}のダウンロードが完了するのを待つ。"

#: cddagl/ui/views/dialogs.py:65
msgid "3. Select the downloaded archive."
msgstr "3. ダウンロードしたアーカイブを選択する。"

#: cddagl/ui/views/dialogs.py:90
msgid "Install this {name}"
msgstr "{name}をインストール"

#: cddagl/ui/views/dialogs.py:96
msgid "Do not install"
msgstr "インストールしない"

#: cddagl/ui/views/dialogs.py:107
msgid "Browser download"
msgstr "ブラウザダウンロード"

#: cddagl/ui/views/dialogs.py:112
msgid "Downloaded archive"
msgstr "アーカイブをダウンロードしました"

#: cddagl/ui/views/dialogs.py:113
msgid "Archive files {formats}"
msgstr "アーカイブファイル {formats}"

#: cddagl/ui/views/dialogs.py:172
msgid "About Kitten CDDA Launcher"
msgstr "About CDDA Game について"

#: cddagl/ui/views/dialogs.py:238
msgid ""
"The Kitten CDDA Launcher just crashed. An unhandled exception was raised. Here"
" are the details."
msgstr "不明な例外処理が発生し、The Kitten CDDA Launcher は動作を停止しました。詳細はこちら。"

#: cddagl/ui/views/dialogs.py:288
msgid "Exit"
msgstr "終了"

#: cddagl/ui/views/dialogs.py:294
msgid "Something went wrong"
msgstr "何かが間違っています"

#: cddagl/ui/views/main.py:159 cddagl/ui/views/main.py:169
#: cddagl/ui/views/main.py:677 cddagl/ui/views/main.py:678
#: cddagl/ui/views/main.py:768 cddagl/ui/views/main.py:810
#: cddagl/ui/views/main.py:940 cddagl/ui/views/main.py:1049
#: cddagl/ui/views/main.py:1139 cddagl/ui/views/main.py:1248
#: cddagl/ui/views/main.py:2671 cddagl/ui/views/main.py:2769
#: cddagl/ui/views/main.py:2867 cddagl/ui/views/soundpacks.py:1013
#: cddagl/ui/views/soundpacks.py:1049
msgid "Unknown"
msgstr "不明"

#: cddagl/ui/views/main.py:206
msgid "Directory:"
msgstr "ディレクトリ:"

#: cddagl/ui/views/main.py:207
msgid "Version:"
msgstr "バージョン:"

#: cddagl/ui/views/main.py:208
msgid "Build:"
msgstr "ビルド:"

#: cddagl/ui/views/main.py:209
msgid "Saves:"
msgstr "セーブ:"

#: cddagl/ui/views/main.py:211
msgid ""
"Your save directory might be large enough to cause significant delays during the update process.\n"
"You might want to enable the \"Do not copy or move the save directory\" option in the settings tab."
msgstr ""
"アップデートプロセス中の重要な遅延のため、セーブディレクトリは十分な領域を必要とします。\n"
"設定タブから \"セーブディレクトリをコピー・移動しない\" 設定ができます。"

#: cddagl/ui/views/main.py:215 cddagl/ui/views/main.py:518
#: cddagl/ui/views/main.py:897
msgid "Launch game"
msgstr "ゲーム起動"

#: cddagl/ui/views/main.py:216
msgid "Restore previous version"
msgstr "以前のバージョンを復元する"

#: cddagl/ui/views/main.py:217
msgid "Game"
msgstr "ゲーム"

#: cddagl/ui/views/main.py:379
msgid "before_launch"
msgstr "起動前"

#: cddagl/ui/views/main.py:391
msgid "Game executable not found"
msgstr "実行ファイルがありません"

#: cddagl/ui/views/main.py:413
msgid "Could not launch the game executable"
msgstr "実行ファイルの起動に失敗しました"

#: cddagl/ui/views/main.py:416
msgid "Cannot launch game"
msgstr "起動出来ませんでした"

#: cddagl/ui/views/main.py:418
msgid ""
"\n"
"<p>The launcher failed to start the game executable in <strong>{filename}</strong> .</p>\n"
"<p>It received the following error from the operating system: {error}</p>\n"
"<p>Poor antivirus products are known to detect the game binary as a threat and\n"
"block its execution. A simple workaround is to add the game binary in your\n"
"antivirus whitelist or select the action to trust this binary when detected.</p>\n"
msgstr ""
"\n"
"<p>ゲーム実行ファイルの起動に失敗しました。:<strong>{filename}</strong></p>\n"
"<p>システムから以下のエラーを受け取りました。: {error}</p>\n"
"<p>一部のアンチウイルス製品はゲームバイナリを脅威と誤認し、その起動をブロックしてし\n"
"まうことがあります。ゲームバイナリをホワイトリストに登録するか、ブロックされた際に「信頼す\n"
"る」を選択することで回避できるかもしれません。\n"

#: cddagl/ui/views/main.py:425
msgid "[unknown]"
msgstr "[不明]"

#: cddagl/ui/views/main.py:444 cddagl/ui/views/main.py:757
#: cddagl/ui/views/main.py:845 cddagl/ui/views/main.py:2656
#: cddagl/ui/views/main.py:2952
msgid "Game process is running"
msgstr "ゲーム実行中"

#: cddagl/ui/views/main.py:462 cddagl/ui/views/main.py:863
msgid "Show current game"
msgstr "現在のゲームを表示"

#: cddagl/ui/views/main.py:500 cddagl/ui/views/main.py:887
msgid "Game process has ended"
msgstr "ゲーム実行終了"

#: cddagl/ui/views/main.py:528 cddagl/ui/views/main.py:907
msgid "after_end"
msgstr "終了後"

#: cddagl/ui/views/main.py:586
msgid "Game directory"
msgstr "ゲームディレクトリ"

#: cddagl/ui/views/main.py:642 cddagl/ui/views/main.py:1041
msgid "console"
msgstr "コンソール"

#: cddagl/ui/views/main.py:645 cddagl/ui/views/main.py:1044
msgid "tiles"
msgstr "タイル"

#: cddagl/ui/views/main.py:671 cddagl/ui/views/main.py:2529
#: cddagl/ui/views/main.py:2812 cddagl/ui/views/main.py:2884
msgid "Install game"
msgstr "インストール"

#: cddagl/ui/views/main.py:684 cddagl/ui/views/main.py:1305
#: cddagl/ui/views/main.py:2527 cddagl/ui/views/main.py:2795
#: cddagl/ui/views/main.py:2882
msgid "Update game"
msgstr "アップデート"

#: cddagl/ui/views/main.py:716 cddagl/ui/views/main.py:1088
msgid "Reading: {0}"
msgstr "読み込み中: {0}"

#: cddagl/ui/views/main.py:752
msgid "Previous version restored"
msgstr "以前のバージョンが復元されました"

#: cddagl/ui/views/main.py:754 cddagl/ui/views/main.py:2651
#: cddagl/ui/views/main.py:2949 cddagl/ui/views/tabbed.py:85
msgid "Ready"
msgstr "準備完了"

#: cddagl/ui/views/main.py:804 cddagl/ui/views/main.py:2499
#: cddagl/ui/views/main.py:2807
msgid "Your game is up to date"
msgstr "ゲームは最新版です"

#: cddagl/ui/views/main.py:806 cddagl/ui/views/main.py:2501
#: cddagl/ui/views/main.py:2809
msgid "There is a new update available"
msgstr "アップデート可能です"

#: cddagl/ui/views/main.py:949 cddagl/ui/views/main.py:987
#: cddagl/ui/views/main.py:1014
msgid "World"
msgid_plural "Worlds"
msgstr[0] "ワールド"

#: cddagl/ui/views/main.py:950 cddagl/ui/views/main.py:988
#: cddagl/ui/views/main.py:1015
msgid "Character"
msgid_plural "Characters"
msgstr[0] "キャラクター"

#: cddagl/ui/views/main.py:1048
msgid "Not a CDDA directory"
msgstr "CDDAディレクトリではありません"

#: cddagl/ui/views/main.py:1060
msgid ""
"No executable found in the downloaded archive. You might want to restore "
"your previous version."
msgstr "ダウンロードしたアーカイブの中に実行ファイルがありません。以前のバージョンを復元することをおすすめします。"

#: cddagl/ui/views/main.py:1299
msgid "Platform:"
msgstr "プラットフォーム:"

#: cddagl/ui/views/main.py:1302
msgid "Available builds:"
msgstr "有効なビルド:"

#: cddagl/ui/views/main.py:1303
msgid "Refresh"
msgstr "更新"

#: cddagl/ui/views/main.py:1304
msgid "Changelog"
msgstr "更新履歴"

#: cddagl/ui/views/main.py:1306 cddagl/ui/views/settings.py:391
msgid "Update/Installation"
msgstr "アップデート/インストール"

#: cddagl/ui/views/main.py:1423
msgid "Game is up to date"
msgstr "ゲームは最新版です"

#: cddagl/ui/views/main.py:1424
msgid "You already have the latest version."
msgstr "すでに最新版がインストールされています。"

#: cddagl/ui/views/main.py:1426
msgid "Are you sure you want to update your game?"
msgstr "アップデートを開始しますか？"

#: cddagl/ui/views/main.py:1428
msgid "Update the game again"
msgstr "もう一度アップデートする"

#: cddagl/ui/views/main.py:1430
msgid "I do not need to update the game again"
msgstr "もう一度アップデートしない"

#: cddagl/ui/views/main.py:1458
msgid "Cannot install game on a file"
msgstr "ファイルへのインストールが出来ません"

#: cddagl/ui/views/main.py:1498 cddagl/ui/views/main.py:1520
#: cddagl/ui/views/main.py:1550 cddagl/ui/views/main.py:1575
#: cddagl/ui/views/main.py:1598
msgid "Update cancelled"
msgstr "アップデートが中止されました"

#: cddagl/ui/views/main.py:1501 cddagl/ui/views/main.py:1523
#: cddagl/ui/views/main.py:1553 cddagl/ui/views/main.py:1578
#: cddagl/ui/views/main.py:1601
msgid "Installation cancelled"
msgstr "インストールが中止されました"

#: cddagl/ui/views/main.py:1706 cddagl/ui/views/main.py:1771
#: cddagl/ui/views/soundpacks.py:372 cddagl/ui/views/soundpacks.py:567
msgid "Downloading: {0}"
msgstr "ダウンロード中: {0}"

#: cddagl/ui/views/main.py:1742 cddagl/ui/views/tabbed.py:505
msgid "Cancel update"
msgstr "アップデート中止"

#: cddagl/ui/views/main.py:1744
msgid "Cancel installation"
msgstr "インストール中止"

#: cddagl/ui/views/main.py:1810 cddagl/ui/views/soundpacks.py:440
#: cddagl/ui/views/soundpacks.py:603
msgid "Testing downloaded file archive"
msgstr "ダウンロード済みのアーカイブをテスト中"

#: cddagl/ui/views/main.py:1847 cddagl/ui/views/soundpacks.py:487
#: cddagl/ui/views/soundpacks.py:649
msgid "Downloaded archive is invalid"
msgstr "ダウンロード済みのアーカイブが無効です"

#: cddagl/ui/views/main.py:1857
msgid "Could not download game"
msgstr "ダウンロード出来ませんでした"

#: cddagl/ui/views/main.py:1886 cddagl/ui/views/main.py:1892
#: cddagl/ui/views/main.py:2465
msgid "previous_version directory"
msgstr "以前のバージョンのディレクトリ"

#: cddagl/ui/views/main.py:1930
msgid "Backing up current game"
msgstr "現在のゲームをバックアップ中"

#: cddagl/ui/views/main.py:1974
msgid "Backing up {0}"
msgstr "バックアップ中 {0}"

#: cddagl/ui/views/main.py:2076
#: cddagl/ui/views/soundpacks.py:783
msgid "Extracting {0}"
msgstr "展開中 {0}"

#: cddagl/ui/views/main.py:2086
msgid "Cannot extract game archive"
msgstr "展開に失敗しました"

#: cddagl/ui/views/main.py:2088
msgid ""
"\n"
"<p>The launcher failed to extract the game archive.</p>\n"
"<p>It received the following error from the operating system: {error}</p>"
msgstr ""
"\n"
"<p>アーカイブの展開に失敗しました。</p>\n"
"<p>システムから以下のエラーを受け取りました。: {error}</p>"

#: cddagl/ui/views/main.py:2165
msgid "{0} directory"
msgstr "{0} ディレクトリ"

#: cddagl/ui/views/main.py:2222
msgid "Restoring custom tilesets"
msgstr "カスタムタイルセットを復元中"

#: cddagl/ui/views/main.py:2265
msgid "Restoring custom soundpacks"
msgstr "カスタムサウンドパックを復元中"

#: cddagl/ui/views/main.py:2314
msgid "{name} soundpack"
msgstr "{name} サウンドパック"

#: cddagl/ui/views/main.py:2344
msgid "Restoring custom mods"
msgstr "カスタムModを復元中"

#: cddagl/ui/views/main.py:2431
msgid "Restoring custom fonts"
msgstr "カスタムフォントを復元中"

#: cddagl/ui/views/main.py:2486
msgid "Update completed"
msgstr "アップデート完了"

#: cddagl/ui/views/main.py:2488
msgid "Installation completed"
msgstr "インストール完了"

#: cddagl/ui/views/main.py:2571
msgid "Fetching remote builds"
msgstr "リモートビルド取り込み中"

#: cddagl/ui/views/main.py:2577 cddagl/ui/views/main.py:2619
msgid "Fetching: {url}"
msgstr "取り込み中: {url}"

#: cddagl/ui/views/main.py:2818
msgid "Could not find remote builds"
msgstr "リモートビルドがありません"

#: cddagl/ui/views/main.py:2907
msgid "<h3>Loading changelog...</h3>"
msgstr "<h3>更新履歴を読み込み中...</h3>"

#: cddagl/ui/views/main.py:2955
msgid "<h3>Parsing changelog...</h3>"
msgstr "<h3>更新履歴を解析中...</h3>"

#: cddagl/ui/views/main.py:3140 cddagl/ui/views/main.py:3377
#, python-format
msgid "Source path '%s' is not a directory"
msgstr "ソースの場所 '%s' はディレクトリではありません"

#: cddagl/ui/views/main.py:3171 cddagl/ui/views/main.py:3420
msgid "file"
msgid_plural "files"
msgstr[0] "ファイル"

#: cddagl/ui/views/main.py:3173
msgid "Analysing {name} - Found {file_count} {files}"
msgstr "解析中 {name} - 検出 {file_count} {files}"

#: cddagl/ui/views/main.py:3218 cddagl/ui/views/main.py:3282
msgid "Cannot remove directory"
msgstr "ディレクトリを削除できません"

#: cddagl/ui/views/main.py:3225 cddagl/ui/views/main.py:3288
msgid ""
"\n"
"<p>The launcher failed to remove the following directory: {directory}</p>\n"
"<p>When trying to remove or access {filename}, the launcher raised the\n"
"following error: {error}</p>"
msgstr ""
"\n"
"<p>以下のディレクトリの削除に失敗しました: {directory}</p>\n"
"<p>{filename}へのアクセスを試みましたがエラーが発生しました\n"
": {error}</p>"

#: cddagl/ui/views/main.py:3234 cddagl/ui/views/main.py:3297
msgid ""
"\n"
"<p>No process seems to be using that file or directory.</p>"
msgstr ""
"\n"
"<p>選択されたファイルまたはディレクトリを使用中のプロセスはありません。</p>"

#: cddagl/ui/views/main.py:3237 cddagl/ui/views/main.py:3300
msgid ""
"\n"
"<p>The process <strong>{image_file_name} ({pid})</strong> is currently using\n"
"that file or directory. You might need to end it if you want to retry.</p>"
msgstr ""
"\n"
"<p>プロセス <strong>{image_file_name} ({pid})</strong> が、指定された\n"
"ファイルまたはディレクトリを使用中です。 上記プロセス終了後に再試行して下さい。</p>"

#: cddagl/ui/views/main.py:3245 cddagl/ui/views/main.py:3307
msgid "Do you want to retry removing this directory?"
msgstr "ディレクトリの削除を再試行しますか？"

#: cddagl/ui/views/main.py:3248 cddagl/ui/views/main.py:3310
msgid "Retry removing the directory"
msgstr "ディレクトリの削除を再試行する"

#: cddagl/ui/views/main.py:3329
msgid "Deleting {name} - {entry}"
msgstr "削除中 {name} - {entry}"

#: cddagl/ui/views/main.py:3339 cddagl/ui/views/main.py:3551
msgid "Analysing {name}"
msgstr "解析中 {name}"

#: cddagl/ui/views/main.py:3379
#, python-format
msgid "Destination path '%s' already exists"
msgstr "実行パス '%s' はすでに存在します"

#: cddagl/ui/views/main.py:3423
msgid "Analysing {name} - Found {file_count} {files} ({size})"
msgstr "解析中 {name} - 検出 {file_count} {files} ({size})"

#: cddagl/ui/views/main.py:3541
msgid "Copying {name} - {entry}"
msgstr "コピー中 {name} - {entry}"

#: cddagl/ui/views/soundpacks.py:196
msgid "Installed"
msgstr "インストール完了"

#: cddagl/ui/views/soundpacks.py:197
#: cddagl/ui/views/soundpacks.py:896 cddagl/ui/views/soundpacks.py:952
msgid "Disable"
msgstr "無効"

#: cddagl/ui/views/soundpacks.py:198
msgid "Delete"
msgstr "削除"

#: cddagl/ui/views/soundpacks.py:448 cddagl/ui/views/soundpacks.py:493
#: cddagl/ui/views/soundpacks.py:611 cddagl/ui/views/soundpacks.py:655
msgid "Selected file is a bad archive file"
msgstr "選択されたアーカイブファイルが壊れています"

#: cddagl/ui/views/soundpacks.py:455 cddagl/ui/views/soundpacks.py:618
msgid "Selected file is a password protected archive file"
msgstr "選択されたアーカイブにはパスワードがかかっています"

#: cddagl/ui/views/soundpacks.py:528
msgid "Soundpack installation cancelled"
msgstr "サウンドパックのインストールを中止しました"

#: cddagl/ui/views/soundpacks.py:475
#: cddagl/ui/views/soundpacks.py:637
msgid "Unknown downloaded archive format ({extension})"
msgstr "不明な圧縮フォーマットです ({extension})"

#: cddagl/ui/views/soundpacks.py:880 cddagl/ui/views/soundpacks.py:1098
msgid " (Disabled)"
msgstr "(無効)"

#: cddagl/ui/views/soundpacks.py:881 cddagl/ui/views/soundpacks.py:954
msgid "Enable"
msgstr "有効"

#: cddagl/ui/views/soundpacks.py:994
msgid "Getting remote size"
msgstr "リモートサイズ取得中"

#: cddagl/ui/views/settings.py:96 cddagl/ui/views/settings.py:162
msgid "System language or best match ({locale})"
msgstr "システム言語または推奨 ({locale})"

#: cddagl/ui/views/settings.py:157
msgid "Command line parameters:"
msgstr "コマンドラインパラメータ:"

#: cddagl/ui/views/settings.py:159
msgid "Keep the launcher opened after launching the game"
msgstr "ゲーム起動後にランチャーを閉じない"

#: cddagl/ui/views/settings.py:160
msgid "Language:"
msgstr "言語:"

#: cddagl/ui/views/settings.py:164
msgid "Allow multiple instances of the launcher to be started"
msgstr "複数起動を許可する"

#: cddagl/ui/views/settings.py:167
msgid "Use the launcher directory as the game directory"
msgstr "ランチャーが存在するディレクトリをゲームティレクトリとする"

#: cddagl/ui/views/settings.py:169 cddagl/ui/views/tabbed.py:243
msgid "Do not check for new version of the Kitten CDDA Launcher on launch"
msgstr "起動時に新しいバージョンをチェックしない"

#: cddagl/ui/views/settings.py:171
msgid "Launcher"
msgstr "ランチャー"

#: cddagl/ui/views/settings.py:374
msgid "Do not copy or move the save directory"
msgstr "セーブディレクトリをコピー・移動しない"

#: cddagl/ui/views/settings.py:376
msgid ""
"If your save directory size is large, it might take a long time to copy it during the update process.\n"
"This option might help you speed the whole thing but your previous version will lack the save directory."
msgstr ""
"セーブディレクトリが巨大になると、アップデートの際それをコピーするために長時間を要します。\n"
"この設定を有効にすることで高速化できますが、以前のバージョンのセーブディレクトリは欠落します。"

#: cddagl/ui/views/settings.py:381
msgid "Keep a copy of the downloaded archive in the following directory:"
msgstr "ダウンロードしたアーカイブのコピーを保存するディレクトリ:"

#: cddagl/ui/views/settings.py:384
msgid "Automatically refresh builds list every"
msgstr "ビルド一覧の自動更新間隔"

#: cddagl/ui/views/settings.py:385
msgid "minutes"
msgstr "分"

#: cddagl/ui/views/settings.py:386
msgid "Remove previous version after update (not recommended)"
msgstr "アップデート後に以前のバージョンを削除する(非推奨)"

#: cddagl/ui/views/settings.py:443
msgid "Archive directory"
msgstr "アーカイブディレクトリ"

#: cddagl/ui/views/soundpacks.py:201
msgid "Add this new soundpack to the repository"
msgstr "この新しいサウンドパックをリポジトリに追加する"

#: cddagl/ui/views/soundpacks.py:202
msgid ""
"* Name: [Enter the name of the soundpack]\n"
"* Url: [Enter the Url where we can find the soundpack]\n"
"* Author: [Enter the name of the author]\n"
"* Homepage: [Enter the Url of the author website or where the soundpack was published]\n"
"* Soundpack not found in version: {version}\n"
msgstr ""
"* Name: [サウンドパックの名前]\n"
"* Url: [サウンドパックをダウンロード可能なURL]\n"
"* Author: [作者の名前]\n"
"* Homepage: [作者の公式サイトまたは配布されている場所のURL]\n"
"* Soundpack not found in version: {version}\n"

#: cddagl/ui/views/soundpacks.py:210
msgid "<a href=\"{url}\">Suggest a new soundpack on GitHub</a>"
msgstr "<a href=\"{url}\">GitHubで新しいサウンドパックを提案する</a>"

#: cddagl/ui/views/soundpacks.py:212 cddagl/ui/views/soundpacks.py:671
msgid "Install this soundpack"
msgstr "このサウンドパックをインストールする"

#: cddagl/ui/views/soundpacks.py:214
msgid "View name:"
msgstr "表示名:"

#: cddagl/ui/views/soundpacks.py:321
msgid "Soundpack already present"
msgstr "サウンドパックがすでに存在しています"

#: cddagl/ui/views/soundpacks.py:323
msgid ""
"It seems this soundpack is already installed. The launcher will not "
"overwrite the soundpack if it has the same directory name. You might want to"
" delete the soundpack first if you want to update it. Also, there can only "
"be a single soundpack with the same name value available in the game."
msgstr ""
"このサウンドパックはすでにインストールされているようです。同じ名前のディレクトリが存在する場合、本ランチャーはサウンドパックを上書きしません。アップデートする場合はまずサウンドパックを削除してください。同じ名前のサウンドパックはゲーム中で1つしか有効になりません。"

#: cddagl/ui/views/soundpacks.py:330
msgid "Are you sure you want to install the {view} soundpack?"
msgstr "サウンドパック {view} をインストールしますか？"

#: cddagl/ui/views/soundpacks.py:333
msgid "Install the soundpack"
msgstr "サウンドパックをインストール"

#: cddagl/ui/views/soundpacks.py:407 cddagl/ui/views/soundpacks.py:426
msgid "Cancel soundpack installation"
msgstr "サウンドパックのインストールを中止"

#: cddagl/ui/views/soundpacks.py:812
msgid "Finding the soundpack"
msgstr "サウンドパックを探しています"

#: cddagl/ui/views/soundpacks.py:840
msgid ""
"Soundpack installation cancelled - There is no soundpack in the downloaded "
"archive"
msgstr "サウンドパックのインストールを中止しました - ダウンロードしたアーカイブの中にサウンドパックが存在しません"

#: cddagl/ui/views/soundpacks.py:850
msgid ""
"Soundpack installation cancelled - There is already a {basename} directory "
"in {soundpacks_dir}"
msgstr "サウンドパックのインストールを中止しました - すでに {basename} が {soundpacks_dir} に存在しています"

#: cddagl/ui/views/soundpacks.py:856
msgid "Soundpack installation completed"
msgstr "サウンドパックのインストールが完了しました"

#: cddagl/ui/views/soundpacks.py:912
msgid "Delete soundpack"
msgstr "サウンドパックを削除"

#: cddagl/ui/views/soundpacks.py:913
msgid "This will delete the soundpack directory. It cannot be undone."
msgstr "サウンドパックをディレクトリごと削除します。この操作は元に戻せません。"

#: cddagl/ui/views/soundpacks.py:915
msgid "Are you sure you want to delete the {view} soundpack?"
msgstr "サウンドパック {view} を削除してよろしいですか?"

#: cddagl/ui/views/soundpacks.py:917
msgid "Delete the soundpack"
msgstr "サウンドパックを削除"

#: cddagl/ui/views/soundpacks.py:919
msgid "I want to keep the soundpack"
msgstr "サウンドパックを削除しない"

#: cddagl/ui/views/soundpacks.py:928
msgid "Soundpack deletion cancelled"
msgstr "サウンドパックの削除を中止しました"

#: cddagl/ui/views/soundpacks.py:933
msgid "Soundpack deleted"
msgstr "サウンドパックを削除しました"

#: cddagl/ui/views/tabbed.py:70 cddagl/ui/views/tabbed.py:93
msgid "&File"
msgstr "ファイル"

#: cddagl/ui/views/tabbed.py:71 cddagl/ui/views/tabbed.py:97
msgid "E&xit"
msgstr "終了"

#: cddagl/ui/views/tabbed.py:72 cddagl/ui/views/tabbed.py:101
msgid "&Help"
msgstr "ヘルプ"

#: cddagl/ui/views/tabbed.py:74 cddagl/ui/views/tabbed.py:106
msgid "&Check for update"
msgstr "アップデート確認"

#: cddagl/ui/views/tabbed.py:75 cddagl/ui/views/tabbed.py:112
msgid "&About Kitten CDDA Launcher"
msgstr "&Kitten CDDA Launcher について"

#: cddagl/ui/views/tabbed.py:254
msgid "Launcher update"
msgstr "アップデート"

#: cddagl/ui/views/tabbed.py:255
msgid ""
"You are using version {version} but there is a new update for CDDA Game "
"Launcher. Would you like to update?"
msgstr "使用中の {version} よりも新しいバージョンがあります。アップデートしますか？"

#: cddagl/ui/views/tabbed.py:260
msgid "Update the launcher"
msgstr "アップデートする"

#: cddagl/ui/views/tabbed.py:262
msgid "Not right now"
msgstr "後で"

#: cddagl/ui/views/tabbed.py:293
msgid "Up to date"
msgstr "最新版"

#: cddagl/ui/views/tabbed.py:294
msgid "The Kitten CDDA Launcher is up to date."
msgstr "お使いの Kitten CDDA Launcher は最新版です。"

#: cddagl/ui/views/tabbed.py:401 cddagl/ui/views/tabbed.py:419
msgid "Main"
msgstr "メイン"

#: cddagl/ui/views/tabbed.py:402 cddagl/ui/views/tabbed.py:424
msgid "Backups"
msgstr "バックアップ"

#: cddagl/ui/views/tabbed.py:403 cddagl/ui/views/tabbed.py:429
msgid "Mods"
msgstr "Mod"

#: cddagl/ui/views/tabbed.py:405 cddagl/ui/views/tabbed.py:439
msgid "Soundpacks"
msgstr "サウンドパック"

#: cddagl/ui/views/tabbed.py:407 cddagl/ui/views/tabbed.py:449
msgid "Settings"
msgstr "設定"

#: cddagl/ui/views/tabbed.py:434
msgid "Tilesets"
msgstr "タイルセット"

#: cddagl/ui/views/tabbed.py:444
msgid "Fonts"
msgstr "フォント"

#: cddagl/ui/views/tabbed.py:467
msgid "Progress:"
msgstr "進捗:"

#: cddagl/ui/views/tabbed.py:496
msgid "Speed:"
msgstr "速度:"

#: cddagl/ui/views/tabbed.py:515
msgid "Kitten CDDA Launcher self-update"
msgstr "Kitten CDDA Launcher セルフアップデート"
